src/swarm-engine/Swarm/Game/ScenarioInfo.hs

```diff
@@ -128,17 +143,16 @@ normalizeScenarioPath col p =
         then return path
         else liftIO $ do
           canonPath <- canonicalizePath path
-          eitherDdir <- runM . runThrow @SystemFailure $ getDataDirSafe Scenarios "." -- no way we got this far without data directory
-          d <- canonicalizePath $ fromRight' eitherDdir
+          eitherDataDir <- runM . runThrow @SystemFailure $ getDataDirSafe Scenarios "." -- no way we got this far without data directory
+          d <- canonicalizePath $ fromRight' eitherDataDir
           let n =
                 stripPrefix (d </> "scenarios") canonPath
                   & maybe canonPath (dropWhile (== pathSeparator))
           return n
 
 -- | Convert a scenario collection to a list of scenario items.
 scenarioCollectionToList :: ScenarioCollection -> [ScenarioItem]
-scenarioCollectionToList (SC Nothing m) = M.elems m
-scenarioCollectionToList (SC (Just order) m) = (m M.!) <$> order
+scenarioCollectionToList (SC xs) = map snd $ OM.assocs xs
```

Comment:
Is there not an `OM.elems` function or something similar?

Reply:
Weirdly not.  Defined a local util function for it.